Makefile

```diff
@@ -2,6 +2,18 @@
 DESTDIR=/usr/local
 PREFIX=mbedtls_
 
+PROGRAMS_DIR=./programs
+TESTS_DIR=./tests
+
+DIR_FOR_MBED_TLS_ENV=./library
```

Comment:
Could you add a comment that this is for auto-detection of Mbed TLS builds within Mbed OS?